config.lua.dist

```diff
@@ -161,6 +161,10 @@ removeOnDespawn = true
 walkToSpawnRadius = 15
 monsterOverspawn = false
 
+-- Bestiary
+bestiaryPartyKillSharing = false
+bestiaryKillsPoints = 1
```

Comment:
```suggestion
bestiaryPointsPerKill = 1
```

Reply:
the party kill sharing I agree, but multiple points per kill is a very popular event, but I'm fine with either outcome